cookbook/validation/severity.rst

```diff
@@ -0,0 +1,165 @@
+             */
+            protected $bankAccountNumber;
+        }
+
+    .. code-block:: xml
+
+        <!-- src/AppBundle/Resources/config/validation.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="AppBundle\Entity\User">
+                <property name="username">
+                    <constraint name="NotBlank">
+                        <option name="payload">
+                            <value key="severity">error</value>
+                        </option>
+                    </constraint>
+                </property>
+                <property name="password">
+                    <constraint name="NotBlank">
+                        <option name="payload">
+                            <value key="severity">error</value>
+                        </option>
+                    </constraint>
+                </property>
+                <property name="bankAccountNumber">
+                    <constraint name="Iban">
+                        <option name="payload">
+                            <value key="severity">warning</value>
+                        </option>
+                    </constraint>
+                </property>
+            </class>
+        </constraint-mapping>
+
+    .. code-block:: php
+
+        // src/AppBundle/Entity/User.php
+        namespace AppBundle\Entity;
+
+        use Symfony\Component\Validator\Mapping\ClassMetadata;
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class User
+        {
+            public static function loadValidatorMetadata(ClassMetadata $metadata)
+            {
+                $metadata->addPropertyConstraint('username', new Assert\NotBlank(array(
+                    'payload' => array('severity' => 'error'),
+                )));
+                $metadata->addPropertyConstraint('password', new Assert\NotBlank(array(
+                    'payload' => array('severity' => 'error'),
+                )));
+                $metadata->addPropertyConstraint('bankAccountNumber', new Assert\Iban(array(
+                    'payload' => array('severity' => 'warning'),
+                )));
+            }
+        }
+
+2. Customize the Error Message Template
+---------------------------------------
+
+.. versionadded:: 2.6
+    The ``getConstraint()`` method in the ``ConstraintViolation`` class was
+    introduced in Symfony 2.6.
+
+When validating the ``User`` object failed, you can retrieve the constraint
+that caused a particular failure using the
+:method:`Symfony\\Component\\Validator\\ConstraintViolation::getConstraint`
+method. Each constraint exposes the attached payload as a public property::
```

Comment:
The double colon is a shortcut for the default language so that you don't need to add a `code-block` directive.